### PR TITLE
[Method References] LPS-162833 Convert method references to field references in module portal-search-elasticsearch7-impl | part 3

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I convert method references to field references in module portal-search-elasticsearch7-impl.

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results ? Thanks!

**Comment by catarineaguiar** [comment]
@W2jx no regressions found.

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchIndexSearcher.java [hunk 1]
```diff
@@ -235,10 +235,8 @@ public long searchCount(SearchContext searchContext, Query query) {
 		}
 	}
 
-	@Override
-	@Reference(target = "(search.engine.impl=Elasticsearch)", unbind = "-")
 	public void setQuerySuggester(QuerySuggester querySuggester) {
-		super.setQuerySuggester(querySuggester);
+		_querySuggester = querySuggester;
```
Comment (tinatian) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchIndexSearcher.java:239]: @W2jx This setQuerySuggester is only used in test, you should remove it and fix the test. You should never to have any code just for a test exposed.

Tina

**Comment by tinatian** [comment]
@W2jx Please see my comment and resend.

Tina

**Pull request closed without merging**